use `useTheme` from nextui instead of next-themes

erenkulaksiz opened the pull request:
## [LEVEL]/[COMPONENT]
**TASK**: Potential fix for https://github.com/nextui-org/nextui/pull/200

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### Description, Motivation and Context
Instead of getting isDark state from next-themes, it will be better to get it from next-ui to sync up with theme. Also this might fix the pull request i put above.

### Screenshots - Animations

jrgarciadev commented:
Hey @erenkulaksiz could you please also apply this to the `theme-switch` component? thanks 🙌🏼.

erenkulaksiz commented:
Done 👍

jrgarciadev commented:
Hey @erenkulaksiz  could you please resolve the comment?, you should remove theme `theme` object from  `const { setTheme, theme } = useNextTheme();` , before sending the commit please run `yarn build:docs`, thanks 🙏🏻

erenkulaksiz commented:
> Hey @erenkulaksiz could you please resolve the comment?, you should remove theme `theme` object from `const { setTheme, theme } = useNextTheme();` , before sending the commit please run `yarn build:docs`, thanks 🙏🏻

Hey, sorry for the inconvenience. I fixed it and checked, https://github.com/nextui-org/nextui/pull/200 issue is now fixed.

erenkulaksiz commented on apps/docs/src/components/theme-toggle/index.tsx:
```diff
@@ -29,7 +29,7 @@ const StyledButton = styled('button', {
 export const ThemeToggle: React.FC<Props> = ({ className, css }) => {
   const isMounted = useIsMounted();
   const { setTheme, theme } = useNextTheme();
```
Please try pulling newest commit, i can build docs succesfully locally. Thanks 🙌
